src/Algebra/Construct/Sub/AbelianGroup.agda

```diff
@@ -0,0 +1,78 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Subgroups of Abelian groups: necessarily Normal
+--
+-- This is a strict addition/extension to Nathan van Doorn (Taneb)'s PR
+-- https://github.com/agda/agda-stdlib/pull/2852
+-- and avoids the lemma `Algebra.NormalSubgroup.abelian⇒subgroup-normal`
+-- introduced in PR https://github.com/agda/agda-stdlib/pull/2854
+-- in favour of the direct definition of the field `normal` below.
+------------------------------------------------------------------------
+
+{-# OPTIONS --safe --cubical-compatible #-}
+
+open import Algebra.Bundles using (AbelianGroup)
+
+module Algebra.Construct.Sub.AbelianGroup {c ℓ} (G : AbelianGroup c ℓ) where
```

Comment:
This is better named: it does construct an `AbelianGroup` out of a (normal) subgroup. This seems like it is in the right place.

Reply:
Yes (but still: sigh).